Modules/trigger.py

```diff
@@ -35,22 +35,47 @@ def __init__(self, bot: pydle.BasicClient, nickname: str, target: str,
         self.identified = identified
 
     @classmethod
-    def from_bot_user(cls, bot: pydle.BasicClient, nickname: str, target: str):
+    def from_bot_user(cls, bot: pydle.BasicClient, nickname: str, target: str)\
+            -> 'Trigger':
```

Comment:
Either cut the line at a comma or not at all (preferrably the latter) please, backslashes hurt